Horatio/Operations/AlertOperation.swift

```diff
@@ -38,8 +38,7 @@ class AlertOperation: Operation {
     // MARK: Initialization
 
     init(presentationContext: UIViewController? = nil) {
-        self.presentationContext = presentationContext ?? UIApplication.shared.keyWindow?.rootViewController
-
+        self.presentationContext = presentationContext
```

Comment:
Ah, there shouldn't be, though. That's what the MutuallyExclusive controller is for, e.g.,

addCondition(MutuallyExclusive<UIViewController>())

Wrapping alert displays in an operation with the above control ensures that only one's on screen at once, and that they all display. With the above change, this alert would just get dropped on the floor instead of being displayed once the current alert is dismissed.